classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
```

Comment:
- [ ] I think student should only be read in the Classroom  `attr_reader :label, :students`